nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/src/test/java/org/apache/nifi/csv/TestCSVRecordReader.java

```diff
@@ -592,6 +585,47 @@ public void testExtraFieldNotInHeader() throws IOException, MalformedRecordExcep
         }
     }
 
```

Comment:
You may want to test with multiple duplicates,  id, name, country, id, name, country.
Also, I assume case doesn't matter, but that is an assumption.
id, name, country, ID, NAME, COUNTRY

Reply:
Agreed, added.

Header Names are case-sensitive, so they're treated as different columns/fields.

Handling of duplicates is different between Apache and Jackson, which the updated tests will show (and I'll add something to the "Allow Duplicate Headers" property description)